flycheck.el

```diff
@@ -11238,6 +11239,22 @@ See URL `http://www.ruby-doc.org/stdlib-2.0.0/libdoc/yaml/rdoc/YAML.html'."
   :modes yaml-mode
   :next-checkers ((warning . cwl)))
 
+(flycheck-def-config-file-var flycheck-yamllintrc yaml-yamllint ".yamllint"
+  :safe #'stringp)
+
+(flycheck-define-checker yaml-yamllint
+  "A YAML syntax checker using YAMLLint.
+See URL `https://github.com/adrienverge/yamllint'."
+  :command (
+            "yamllint" "-f" "parsable" source
+            (config-file "-c" flycheck-yamllintrc))
+  :error-patterns
+  ((error line-start
+          (file-name) ":" line ":" column ": [error] " (message) line-end)
+   (warning line-start
+            (file-name) ":" line ":" column ": [warning] " (message) line-end))
```

Comment:
Does Yamllint support structured output, like JSON or XML? We tend to prefer those over plain text, when available.

Reply:
It doesn't. `parsable` is mode supposed by author to be used for building linters etc.

Reply:
Got it, thanks!